backend/app/api/v1/admin.py

```
@@ -1,114 +1,131 @@
+
+router = APIRouter(prefix="/admin", tags=["admin"])
+
+@router.post("/create_user", response_model=AdminUserOut, status_code=status.HTTP_201_CREATED)
+async def create_user(
+    user_data: AdminCreateUser,
+    admin: User = Depends(require_admin),
+    db: AsyncSession = Depends(get_db)
+):
+    try:
+        user = await admin_service.create_user(db, user_data)
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)